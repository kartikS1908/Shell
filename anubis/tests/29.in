path /bin /usr/bin 
echo pipe | xargs echo long | xargs echo really | xargs echo a | xargs echo is | xargs echo this
exit
