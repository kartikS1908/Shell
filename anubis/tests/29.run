./anubis tests/29.in
