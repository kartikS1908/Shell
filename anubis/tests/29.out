this is a really long pipe
